dapps/kiosk/src/index.css

```diff
@@ -0,0 +1,26 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.tab-title {
+  @apply !px-0 py-2 mr-5 flex-shrink-0 !ring-transparent rounded-none border-t-0 border-l-0 border-r-0 
+  outline-transparent !shadow-none border-transparent hover:!border-primary border-b-2 ui-selected:border-primary  ui-not-selected:bg-transparent;
+}
+
+button {
+  @apply ease-in-out duration-300 rounded border border-transparent py-2 px-4 bg-gray-200;
+}
+
+.btn-primary {
```

Comment:
Would generally recommend just actually using tailwind everywhere instead of this